modules/sd_models.py

```diff
@@ -437,6 +439,10 @@ def set_sd_model(self, v):
 
 def load_model(checkpoint_info=None, already_loaded_state_dict=None):
     from modules import lowvram, sd_hijack
+    # added fix to uninitialized model list
+    if models_loaded is False:

```

Comment:
You wouldn't need an extra `models_loaded` state for this.

```suggestion
    if not checkpoints_list:
```

should suffice.